app-pda/libusbmuxd/libusbmuxd-1.0.10.recipe

```diff
@@ -1,29 +1,33 @@
 SUMMARY="A client library to multiplex connections from and to iOS devices"
 DESCRIPTION="libusbmuxd is a client library to multiplex connections from and \
 to iOS devices by connecting to a socket provided by a usbmuxd daemon."
-HOMEPAGE="http://www.libimobiledevice.org/"
+HOMEPAGE="https://www.libimobiledevice.org/"
 COPYRIGHT="2013-2014 Nikias Bassen
 	2009-2014 Martin Szulecki"
 LICENSE="GNU GPL v2
 	GNU LGPL v2.1"
 REVISION="2"
-SOURCE_URI="http://www.libimobiledevice.org/downloads/libusbmuxd-$portVersion.tar.bz2"
+SOURCE_URI="https://www.libimobiledevice.org/downloads/libusbmuxd-$portVersion.tar.bz2"
 CHECKSUM_SHA256="1aa21391265d2284ac3ccb7cf278126d10d354878589905b35e8102104fec9f2"
 PATCHES="libusbmuxd-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
 
-if [ "$targetArchitecture" != x86_gcc2 ]; then
-	commandBinDir=$binDir
-else
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
 	commandBinDir=$prefix/bin
 fi
 
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	libusbmuxd$secondaryArchSuffix = $portVersion compat >= 1
-	lib:libusbmuxd$secondaryArchSuffix = 4.0.0 compat >= 4
-	cmd:iproxy = $portVersion compat >= 1
+	lib:libusbmuxd$secondaryArchSuffix = $libVersionCompat
+	cmd:iproxy$commandSuffix = $portVersion compat >= 1
```

Comment:
Shouldn't `cmd` be placed before `lib`?

Reply:
Yeah, we try to, but I guess we tend to be less strict here, because we have several recipes for which we only add the `cmd:` entries when we're not building for a secondary arch. So, for these recipes we need to start with the `lib:` entries.
Well, here it's not the case, so I believe the only reason we have put `lib:libusbmuxd` before the `cmd:iproxy` was that the `lib:libusbmuxd` is "more" important than the `cmd:iproxy`.

I think this is OK to keep this as is, and only be strict in new recipes.
That said, we can always swap the `lib:` and the `cmd:` the next time we update this recipe.
BTW, I should have done so when I added the `$commandSuffix`...